src/ViewInstance.js

```diff
@@ -686,6 +713,7 @@ class ViewInstance extends React.Component {
       <Pane
         data-test-instance-details
         defaultWidth={paneWidth}
+        style={{ flex: 'auto' }}
```

Comment:
Is the code below doing the same purpose? If so I suggest using it.
```suggestion
        defaultWidth="fill"
```

Reply:
Nope, it's not. defaultWidth sets the wrong value to flex property and style rewrite it.